src/CMakeLists.txt

```diff
@@ -0,0 +1,31 @@
+cmake_minimum_required(VERSION 3.13)
+
+project(Hawk)
+
+list(APPEND files "exec_monitor")
+
+foreach(filename ${files})
```

Comment:
Could you leave a TODO comment to make it clear that this will have to be improved in the future?